blazar/manager/enforcement.py

```diff
@@ -153,6 +144,35 @@ def remove_lease_exception(self, user_name):
     def setup_usage_enforcement(self, project_name):
         self.initialize_project_allocation(project_name)
 
+    def check_user_max_leases(self, lease_values, resource_type):
+        user_leases = [
+            l for l in db_api.lease_get_all_by_user(lease_values['user_id'])
+            if l['status'] in [status.lease.ACTIVE, status.lease.PENDING]]
+
+        leases_by_resource_type = [
```

Comment:
The name `_by_` in a variable immediately makes me think it's going to be a key/value map ("leases by ID"... "leases by name"). Maybe `leases_matching_resource_type`?

Reply:
done